frontend/common/src/components/form/Select/SelectFieldV2.tsx

```diff
@@ -0,0 +1,205 @@
+import React from "react";
+import { Controller, useFormContext } from "react-hook-form";
+import type { RegisterOptions } from "react-hook-form";
+import ReactSelect, { components } from "react-select";
+import type { NoticeProps, PropsValue, GroupBase, Options } from "react-select";
+import camelCase from "lodash/camelCase";
+import { useIntl } from "react-intl";
+import { InputWrapper } from "../../inputPartials";
+
+export type Option = { value: string | number; label: string };
```

Comment:
> rather than use react-select's more complex Option: {value: string; label: string} format, uses simpler value of string or string[]

This seems to contradict?

Reply:
We still need to track it internally for actually building options, but that pull quote is more about it not saving into form value as Option or Option[] :)